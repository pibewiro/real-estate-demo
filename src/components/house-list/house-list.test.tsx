import { render } from '@testing-library/react';
import HouseList from './house-list';

describe('House List Component', () => {
  it('Should Render House list Component', () => {
    const Component = render(<HouseList />);
    expect(Component.container.firstChild).toMatchSnapshot();
  });
});
