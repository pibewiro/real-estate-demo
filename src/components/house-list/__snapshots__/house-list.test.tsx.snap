// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`House List Component Should Render House list Component 1`] = `
<div
  class="house-list"
/>
`;
